codefresh/pipeline-creator/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -x
+
+export PROJECT="${INPUT_PROJECT}"
+export GITHUB_TOKEN="${INPUT_GITHUB_TOKEN}"
+
+git clone "${INPUT_REPO_URL}"
+cd "${INPUT_REPO_NAME}" || exit 1
```

Comment:
`set -e` will ensure that commands that exit non-zero cause execution to stop. We want the other commands like `git clone` and `git checkout` to also abort execution.